JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -11,7 +11,7 @@ class FruitStore {
     private(set) var fruits: [Fruit: Int] = [:]
     
     private init(initialStock: Int) {
-        for fruit in Fruit.allCases {
+        Fruit.allCases.forEach { fruit in
```

Comment:
후행클로저에 단축 인자이름을 활용해서 간단히 나타낼 수도 있습니다~!
```suggestion
        Fruit.allCases.forEach { self.fruits[$0] = initialStock }
```